zeppelin-web/src/app/notebook/paragraph/paragraph-chart-selector.html

```diff
@@ -41,11 +41,21 @@
             ng-click="setGraphMode('scatterChart', true)"><i class="cf cf-scatter-chart"></i>
     </button>
   </div>
-  <span>
-    <button type="button" class="btn btn-default btn-sm" style="margin-left:10px"
-            tooltip="Download Data as TSV" tooltip-placement="bottom"
-            ng-click="exportToTSV()"><i class="fa fa-download"></i>
+  <span class="btn-group">
+    <button type="button" class="btn btn-default btn-sm"
+            style="margin-left:10px"
+            ng-click="exportToDSV(',')">
+      <i class="fa fa-download"></i>
+    </button>
+    <button type="button" class="btn btn-default btn-sm dropdown-toggle"
```

Comment:
The Button size change when the tooltip comes out because of some bootstrap CSS.

you might need to add a new class to this element, something like that would do:

```
.caretBtn {
    padding-right: 4px !important;
    padding-left: 4px !important;
    width: 20px;
}
```

It will also reduce the width of the caret button
![screen shot 2016-06-14 at 4 28 11 pm](https://cloud.githubusercontent.com/assets/710411/16034473/02f68fec-324d-11e6-9083-d2fb31186b43.png)